package.json

```diff
@@ -18,6 +19,9 @@
     "prepublishOnly": "npm run build",
     "chromatic": "npx chromatic"
   },
+  "engines": {
+    "node": ">=12 <17"
```

Comment:
Warum <17? Bin lokal bei 17.4, scheint aber auch so zu funktionierten.

Reply:
hab ich übernommen von der Nora, glaub da gab es Probleme. Ich schau ob >=12 reicht!

Reply:
Kurz Archäologie betrieben: bei Nora kommt die Webpack-Version von Encore. Die momentane Webpack-Version hat was mit openssl am hut, was bei Node 17 umgeschrieben wurde und nicht rückwärtskompatibel ist. Da wir hier kein Encore haben, ist uns das Wurscht und Node 17 is a go!